scripts/errorCheck.ts

```diff
@@ -31,22 +30,18 @@ fs.readFile("src/compiler/diagnosticMessages.json", "utf-8", (err, data) => {
                     const msg = keys.filter(k => messages[k].code === errCode)[0];
                     messages[msg].seen = true;
                 }
-
-                done();
             });
-        }));
+        });
 
-        async.parallelLimit(tasks, 25, done => {

```

Comment:
without the concurrency limit you might run into issues the maximum number of open file descriptors for this process

Reply:
I've counted 50 possible files. Should be fine, shouldn't it?

Reply:
depends on your system and process settings, a few hundred files should be fine though. but it might fail once the amount of files increases (or other stuff runs in parallel in the same process that also opens lots of files, e.g. if the other check in this file is refactored to read all `*.ts` files asynchronously in parallel)

Reply:
The real problem is just the fact that I forgot that these were callbacks that didn't necessarily execute immediately... but, when I tested it, it worked, so I didn't even pay any mind.

I'm not sure where you got 50; this is scanning all of the error baselines of which there are probably about ~5000.

Reply:
> I'm not sure where you got 50; this is scanning all of the error baselines of which there are probably about ~5000.

I've just searched for files matching ".errors.txt" on the repository using the GitHub interface. Could very well be that it's not showing all files or that more files are generated.